src/services/userService.js

```diff
@@ -87,13 +85,14 @@ class UserService {
   static async findOrCreateUser(_user) {
     try {
       const { email } = _user;
-      if (email.includes('andela.com')) _user.role = 'Manager';
+      if (email.includes('mansur')) _user.role = 'Manager';
```

Comment:
I believe you did this to be able to login with your gmail account however, it's blocking me from login